### ISSUE TITLE
uploading cert fails for VDOMs using non-global admin users

### ISSUE THREAD

**Issue by haught** [fortinet/fortiadc-ingress#7]
The api route `/api/upload/certificate_local` requires the vdom url parameter to be specified in order to work for a user who does **not** have Global Admin privileges and instead only has Read-Write System privileges specified for their profile and assigned to given VDOM. 

Logs:

```
------------------ Response start ------------------
{
"payload": -37
}
------------------ Response end ------------------
test2/nginx-ingress: Do POST url https://10.100.100.22/api/upload/certificate_local
test2/nginx-ingress: Do POST url https://10.100.100.22/api/system_certificate_local_cert_group?vdom=vdom1
------------------ Response start ------------------
{
"status": "Succeeded"
}
------------------ Response end ------------------
test2/nginx-ingress: Do POST url https://10.100.100.22/api/system_certificate_local_cert_group_child_group_member?vdom=vdom1&pkey=test2_nginx-ingress
------------------ Response start ------------------
{
"status": "Failed"
"payload": "Empty value is not allowed."
}
------------------ Response end ------------------
```

As you can see from the logs above the Ingress Controller is missing this "vdom" url parameter for `/api/upload/certificate_local`. In our case it would need to look like 
```
test2/nginx-ingress: Do POST url https://10.100.100.22/api/upload/certificate_local?vdom=vdom1
```

We confirmed this behavior on internal automation using the API for our 3 pairs of FortiADC-2000F and 2 pairs of FortiADC-200F running 6.2.5 - 7.1.1 . The vdom field in the json payload is also required, but I can't see that info in the logs - I assume that is also included.

**Comment by haught** [comment]
Looking further at the logs - `/api/system_scripting/upload` might also be having the same problem but we haven't used that API route ourselves. the -37 error is what we get for the cert upload under the same conditions.
```
------------------ Response start ------------------
{
test2/nginx-ingress: Do POST url https://10.100.100.22/api/system_scripting/upload
"payload": -37
}
------------------ Response end ------------------
```

**Comment by haught** [comment]
Literally we just need ?vdom={vdom} appended to any API call to the LB that specifies a vdom in the payload. That is it, and things work.

**Comment by ytlai** [comment]
Thanks for the reporting.
The fix will be included in FortiADC Ingress Controller 2.0.0.
The ETA is in the end of June.

**Comment by ytlai** [comment]
The issue is fixed in Ingress Controller 2.0.0
https://github.com/fortinet/fortiadc-ingress/blob/main/Release-Notes.md
